aws/resource_aws_codebuild_project.go

```diff
@@ -619,6 +625,10 @@ func flattenAwsCodeBuildProjectSource(source *codebuild.ProjectSource) []interfa
 		m["buildspec"] = *source.Buildspec
 	}
 
+	if source.GitCloneDepth != nil {
+		m["git_clone_depth"] = *source.GitCloneDepth
```

Comment:
Two things:
* If the `source.GitCloneDepth` value is always returned, this will change the TypeSet hash when folks upgrade their AWS provider and it will show a diff. We'll need to implement a state migration if that's the case to prevent the diff.
* We should convert this from int64 to int to match the Terraform schema: `int(aws.Int64Value(source.GitCloneDepth))`